tests/unit/curated_recommendations/test_protocol.py

```diff
@@ -115,3 +121,63 @@ def test_must_include_all_column_counts_failure(self, invalid_responsive_layouts
         """Test that Layout creation fails when responsiveLayouts do not include exactly one layout for column counts 1–4."""
         with pytest.raises(ValidationError):
             Layout(name="Invalid Layout", responsiveLayouts=invalid_responsive_layouts)
+
+
+class TestCuratedRecommendationsRequestsProtocol:
+    """Tests for CuratedRecommendationsRequestsProtocol validations."""
+
+    def test_validate_utc_offset(self):
+        """Test that utcOffset validation works correctly."""
+        # Valid utcOffset values
+        valid_offsets = [0, 5.3, 3, 12, 23]
+        for offset in valid_offsets:
+            request = CuratedRecommendationsRequest(
+                locale="en-US",
+                utcOffset=offset,
+            )
+            assert request.utcOffset == round(offset)
+            request = CuratedRecommendationsRequest(
+                locale="en-US",
+                utc_offset=offset,
+            )
+            assert request.utcOffset == round(offset)
```

Comment:
Make this more consistent with the implementation:
```suggestion
        valid_offsets = [0, 5.5, 3, 12, 23]
        for offset in valid_offsets:
            request = CuratedRecommendationsRequest(
                locale="en-US",
                utcOffset=offset,
            )
            assert request.utcOffset == int(offset)
            request = CuratedRecommendationsRequest(
                locale="en-US",
                utc_offset=offset,
            )
            assert request.utcOffset == int(offset)
```